packages/backend/server/src/core/user/resolver.ts

```diff
@@ -137,3 +140,111 @@ export class UserResolver {
     return { success: true };
   }
 }
+
+@InputType()
+class ListUserInput {
+  @Field(() => Int, { nullable: true, defaultValue: 0 })
+  skip!: number;
+
+  @Field(() => Int, { nullable: true, defaultValue: 20 })
+  first!: number;
+}
+
+@InputType()
+class CreateUserInput {
+  @Field(() => String)
+  email!: string;
+
+  @Field(() => String, { nullable: true })
+  name!: string | null;
+
+  @Field(() => String, { nullable: true })
+  password!: string | null;
+
+  @Field(() => Boolean, { nullable: true, defaultValue: true })
+  requireEmailVerification!: boolean;
+}
+
+@Admin()
+@Resolver(() => UserType)
+export class UserManagementResolver {
+  constructor(
+    private readonly db: PrismaClient,
+    private readonly user: UserService,
+    private readonly crypto: CryptoHelper,
+    private readonly config: Config,
+    private readonly event: EventEmitter
+  ) {}
+
+  @Query(() => [UserType], {
+    description: 'List registered users',
+  })
+  async users(
+    @Args({ name: 'filter', type: () => ListUserInput }) input: ListUserInput
+  ): Promise<UserType[]> {
+    const users = await this.db.user.findMany({
+      select: { ...this.user.defaultUserSelect, password: true },
```

Comment:
I think it is not a good idea to allow viewing passwords in the console
we can allow admin to change passwords, but they should not be allowed to view existing passwords.

Reply:
the password won't be returned, it's only selected to show `hasPassword` flag